loader/loader.go

```diff
@@ -556,6 +559,10 @@ func (l *Loader) Close() {
 	}
 
 	l.stopLoad()
+	err := l.toDB.Close()
+	if err != nil {
+		l.tctx.L().Error("close toDB error", log.ShortError(err))
```

Comment:
in the error message, is `downstream` better than `to`?

Reply:
change [it](https://github.com/pingcap/dm/pull/266/files#diff-b38e43f0fb016af3e8cecd2d12d779ecR569)